DXMainClient/Domain/Multiplayer/TeamStartMapping.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ClientCore;
+using Newtonsoft.Json;
+
+namespace DTAClient.Domain.Multiplayer
+{
+    public class TeamStartMapping
+    {
+        private const char SEPARATOR = ':';
+        private const char LIST_SEPARATOR = ',';
+        private const string INVALID_TEAM_MAPPING_ERROR = "Invalid team mapping \"{0}\"";
+        
+        public const string BLOCK = "-";
+        public static readonly List<string> TEAMS = new List<string>() { BLOCK }.Concat(ProgramConstants.TEAMS).ToList();
+
+        [JsonProperty("t")]
+        public string Team { get; set; }
+
+        [JsonProperty("s")]
+        public int Start { get; set; }
+
+        [JsonIgnore]
+        public bool IsValid => TeamIndex != -1;
+
+        [JsonIgnore]
+        public int TeamIndex => TEAMS.IndexOf(Team);
+
+        [JsonIgnore]
+        public int StartingWaypoint => Start - 1;
+
+        [JsonIgnore]
+        public bool IsBlock => Team == BLOCK;
+
+        /// <summary>
+        /// We write these out in the form A:1 where A is the team and 1 is the starting location on the map
+        /// as it would appear in the map preview.
+        /// </summary>
+        /// <returns></returns>
+        public override string ToString()
+        {
+            return Team;
+        }
```

Comment:
Outdated comment? Also not sure if it should be done via `ToString` considering it now doesn't give exhaustive information about the team start mapping in a programmer sense.